lib/chai/utils/addMethod.js

```diff
@@ -39,6 +40,10 @@ module.exports = function (ctx, name, method) {
     if (old_ssfi && config.includeStack === false)
       flag(this, 'ssfi', ctx[name]);
     var result = method.apply(this, arguments);
-    return result === undefined ? this : result;
+
+    var newAssertion = new chai.Assertion();
```

Comment:
I guess this problem has been shadowed by `global.chai = require('../..');` in all tests...